Webpack fails loading syncano npm

According to the [Javascript Quickstart](http://docs.syncano.com/docs/javascript), you should be able to use syncano npm module with webpack .... however, with a minimal webpack config, webpack throws multiple errors in the build pipeline. 

Perhaps it's my limited knowledge of webpack (second time using it) but it seems not as straightforward as the docs suggest. 

webpack config:

```
var webpack = require('webpack');

module.exports = {
    entry: "./src/app.js",
    output: {
        path: './build',
        filename: "build.js"
    },
    module: {
        loaders: [
            { test: /\.vue$/, loader: "vue-loader" },
        ]
    },
    devtool: '#source-map',
}
```

When running the webpack build, multiple errors are thrown but the first two illustrate the two issues: 
1. required modules usually supplied by node are not included and 
2. syntax errors in package json files (which are in fact valid json)

```
ERROR in ./~/syncano/package.json
Module parse failed: /Users/tremendus/Development/Tremendus/harmoni/client/node_modules/syncano/package.json Line 2: Unexpected token :
You may need an appropriate loader to handle this file type.
| {
|   "name": "syncano",
|   "version": "0.2.6",
|   "description": "A library to intereact with the Syncano API.",
 @ ./~/syncano/src/core.js 8:15-41

ERROR in ./~/syncano/~/request/lib/har.js
Module not found: Error: Cannot resolve module 'fs' in /Users/tremendus/Development/Tremendus/harmoni/client/node_modules/syncano/node_modules/request/lib
 @ ./~/syncano/~/request/lib/har.js 3:9-22
```

So is there some special webpack configuration required  or is it not perhaps as easy to use syncano with webpack as suggested in the docs? Is there a sample or tutorial I'm missing? Can we improve the docs?

--- a/.babelrc
+++ b/.babelrc
@@ -1,0 +1,3 @@
+{
+  "plugins": ["inline-package-json"]
+}

--- a/.gitignore
+++ b/.gitignore
@@ -2,3 +2,4 @@ node_modules
 test/coverage
 .DS_Store
 .idea
+lib

--- a/package.json
+++ b/package.json
@@ -2,11 +2,14 @@
   "name": "syncano",
   "version": "0.3.4",
   "description": "A library to intereact with the Syncano API.",
-  "main": "src/syncano.js",
+  "main": "./lib/syncano.js",
   "author": "Kelly Andrews",
   "license": "ISC",
   "scripts": {
-    "coverage": "CODECLIMATE_REPO_TOKEN=2fc62db4794c74bf7d733c365aed53960c0d7b7f9560674868f0e487ce36f2eb codeclimate-test-reporter < ./test/coverage/lcov.info"
+    "coverage": "CODECLIMATE_REPO_TOKEN=2fc62db4794c74bf7d733c365aed53960c0d7b7f9560674868f0e487ce36f2eb codeclimate-test-reporter < ./test/coverage/lcov.info",
+    "prebuild": "rimraf lib",
+    "build": "babel ./src --out-dir ./lib",
+    "prepublish": "npm run build"
   },
   "repository": {
     "type": "git",
@@ -16,6 +19,8 @@
     "bluebird": "^2.9.30"
   },
   "devDependencies": {
+    "babel": "^5.8.34",
+    "babel-plugin-inline-package-json": "^1.0.1",
     "browserify": "^10.2.4",
     "gulp": "^3.9.0",
     "gulp-bump": "^0.3.1",
@@ -32,6 +37,7 @@
     "jshint-stylish": "^2.0.1",
     "mocha": "^2.2.5",
     "mockery": "^1.4.0",
+    "rimraf": "^2.4.4",
     "should": "^7.0.1",
     "should-promised": "^0.3.0",
     "vinyl-buffer": "^1.0.0",

